### PR TITLE
fixes #16

### LINKED ISSUE #16: strong mode still fails
```dart
: [error] Invalid override. The type of Entry.compareTo ((Entry) → int) is not a subtype of Comparable<dynamic>.compareTo ((dynamic) → int). (package:source_maps/builder.dart, line 80, col 3)
```

### PR DESCRIPTION
**Pull request by dam0vm3nt**

### CODE REVIEW

File: lib/builder.dart [hunk 1]
```diff
@@ -77,7 +77,8 @@ class Entry implements Comparable {
   /// location in the target file. We sort primarily by the target offset
   /// because source map files are encoded by printing each mapping in order as
   /// they appear in the target file.
-  int compareTo(Entry other) {
+  int compareTo(_other) {
```
Comment (kevmoo) [inline lib/builder.dart]: Make `Entry implement Comparable<Entry>` instead?

Comment (kevmoo) [inline lib/builder.dart]: Also: we don't use `_understore` names as parameters.

**Comment by kevmoo** [comment]
CC @nex3

**Comment by nex3** [comment]
I'm not a maintainer for this package.

**Comment by kevmoo** [comment]
CC @sigmundch ?

**Comment by sigmundch** [comment]
lgtm, thank for the fix!

**Pull request merged**